applications/PfemFluidDynamicsApplication/python_scripts/pfem_fluid_dynamics_analysis.py

```diff
@@ -0,0 +1,393 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+#Activate it to import in the gdb path:
+#import sys
+#sys.path.append('/home/cpuigbo/kratos')
+#x = input("stopped to allow debug: set breakpoints and press enter to continue");
+import time as timer
+# Import system python
+import os
+
+# Import kratos core and applications
+import KratosMultiphysics
+import KratosMultiphysics.ExternalSolversApplication
+import KratosMultiphysics.DelaunayMeshingApplication
+import KratosMultiphysics.PfemFluidDynamicsApplication
+import KratosMultiphysics.SolidMechanicsApplication
+
+import DEM_coupled_pfem_fluid_dynamics_analysis as fluid_solution
+import pfem_fluid_solver_analysis as pfem_solver
+
+from analysis_stage import AnalysisStage
```

Comment:
it is more future-proof to do the following:
```suggestion
from KratosMultiphysics.analysis_stage import AnalysisStage
```